projects/packages/jetpack-mu-wpcom/src/features/domain-email-nag/domain-email-nag.php

```diff
@@ -49,6 +49,20 @@ function get_domain_with_unverified_email() {
 	return false;
 }
 
+/**
+ * Bump the MC stat for wpcom-domain-email-nag-shown.
+ *
+ * @param string $domain the domain to bump the stat for.
+ */
+function bump_mc_stats( $domain ) {
+	$pixel_url = sprintf(
+		'http://pixel.wp.com/g.gif?v=wpcom-no-pv&x_%s=%s',
+		'wpcom-domain-email-nag-shown',
```

Comment:
I think we need to use pascal_case instead of kebab-case. Pretty sure invalid names get filtered out.
PCYsg-4S2-p2

Reply:
Maybe this?
`wpcom_frontend_unverified_domain_email_nag_shown`

I like saying frontend because it wasn't initially clear to me what context the nag was shown in.